lib/themes/dosomething/paraneue_dosomething/js/reportback/Reportback.js

```diff
@@ -273,7 +278,28 @@ const Reportback = {
         }
       },
     }).done(function(response) {
-      _this.apiUrl = response.meta.pagination.links.next_uri || null;
+      // If response.meta.pagination.links.next, this is from Rogue.
+      if (response.meta.pagination.links.next) {
+        var splitResponseUrl = response.meta.pagination.links.next.split("&");
```

Comment:
Yeah, if we change it in Rogue let's definitely only change it for this one endpoint!